components/profile/personAvatar/index.test.jsx

```diff
@@ -61,6 +62,35 @@ describe("Person Avatar", () => {
     });
     expect(asFragment()).toMatchSnapshot();
   });
+
+  it("renders a webId if there is no name in the profile", async () => {
+    const nameToRemove = solidClientFns.getStringNoLocale(
+      profileThing,
+      foaf.name
+    );
+    const profileThingWithNameRemoved = solidClientFns.removeStringNoLocale(
+      profileThing,
+      foaf.name,
+      nameToRemove
+    );
+
+    const session = mockSession();
+    const SessionProvider = mockSessionContextProvider(session);
+    const { getByText } = renderWithTheme(
+      <CombinedDataProvider
+        solidDataset={profileDataset}
+        thing={profileThingWithNameRemoved}
+      >
+        <SessionProvider>
+          <PersonAvatar profileIri={profileIri} />
+        </SessionProvider>
+      </CombinedDataProvider>
+    );
+
+    await waitFor(() => {
+      expect(getByText(session.info.webId)).toBeInTheDocument();
```

Comment:
Would be better to assert that the text content of the header with the test id equals the webid, incase something else adds the webid to the page

Reply:
I like that - will update

Reply:
updated